ppr-ui/src/components/tombstone/TombstoneDischarge.vue

```diff
@@ -94,7 +97,12 @@ export default defineComponent({
         return 'No Expiry'
       }),
       statusType: computed((): string => {
-        return getMhrInformation.value?.statusType || 'N/A'
+        const parentReg = getMhRegTableBaseRegs.value?.find(reg => reg.mhrNumber === getMhrInformation.value.mhrNumber)
+        // if no parent statusType, fall back on current statusType
+        const regStatus = parentReg?.statusType || getMhrInformation.value.statusType
+        return regStatus === mhApiStatusTypes.FROZEN
+          ? mhUIStatusTypes.ACTIVE
+          : regStatus[0] + regStatus.toLowerCase().slice(1)
```

Comment:
Sorry, build error is because the enum was capitalized 😁  `MhUIStatusTypes`

As the other comment, you can use `isFrozenMhr` to replace the parent reg lookup and condition. 
ie `return isFrozenMhr.value ?  mhUIStatusTypes.ACTIVE : regStatus[0] + regStatus.toLowerCase().slice(1)`